client/state/sharing/publicize/publicize-actions/actions.js

```diff
@@ -14,22 +17,44 @@ import {
 	PUBLICIZE_SHARE_ACTION_EDIT_FAILURE,
 } from 'state/action-types';
 
-export function fetchPostShareActions( siteId, postId ) {
+export function fetchPostShareActionsScheduled( siteId, postId ) {
 	return ( dispatch ) => {
 		dispatch( {
-			type: PUBLICIZE_SHARE_ACTIONS_REQUEST,
+			type: PUBLICIZE_SHARE_ACTIONS_SCHEDULED_REQUEST,
 			siteId,
 			postId,
 		} );
 
-		return new Promise( ( resolve, reject ) => wpcom.req.get( `/sites/${ siteId }/post/${ postId }/publicize/actions`, ( error, data ) => {
+		return new Promise( ( resolve, reject ) => wpcom.req.get( `/sites/${ siteId }/post/${ postId }/publicize/scheduled`, ( error, data ) => {
 			if ( error || ! data.items ) {
-				dispatch( { type: PUBLICIZE_SHARE_ACTIONS_REQUEST_FAILURE, siteId, postId, error } );
+				dispatch( { type: PUBLICIZE_SHARE_ACTIONS_SCHEDULED_REQUEST_FAILURE, siteId, postId, error } );
 				reject();
 			} else {
 				const actions = {};
 				data.items.forEach( action => ( actions[ action.ID ] = action ) );
-				dispatch( { type: PUBLICIZE_SHARE_ACTIONS_REQUEST_SUCCESS, siteId, postId, actions } );
+				dispatch( { type: PUBLICIZE_SHARE_ACTIONS_SCHEDULED_REQUEST_SUCCESS, siteId, postId, actions } );
+				resolve();
+			}
+		} ) );
+	};
+}
+
+export function fetchPostShareActionsPublished( siteId, postId ) {
+	return ( dispatch ) => {
+		dispatch( {
+			type: PUBLICIZE_SHARE_ACTIONS_PUBLISHED_REQUEST,
+			siteId,
+			postId,
+		} );
+
+		return new Promise( ( resolve, reject ) => wpcom.req.get( `/sites/${ siteId }/post/${ postId }/publicize/published`, ( error, data ) => {
+			if ( error || ! data.items ) {
+				dispatch( { type: PUBLICIZE_SHARE_ACTIONS_PUBLISHED_REQUEST_FAILURE, siteId, postId, error } );
+				reject();
+			} else {
+				const actions = {};
+				data.items.forEach( action => ( actions[ action.ID ] = action ) );
```

Comment:
Map will not work since it's not changing the keys and that's all what we do here - change the keys